apiserver/apiserver.go

```diff
@@ -1174,3 +1168,17 @@ func (srv *Server) GetAuditConfig() auditlog.Config {
 	// Delegates to the getter passed in.
 	return srv.getAuditConfig()
 }
+
+// monitoredHandler wraps an HTTP handler for tracking metrics with given labels.
+// It increments and decrements connection counters for monitoring purposes.
+func (srv *Server) monitoredHandler(handler http.Handler, labels ...string) http.Handler {
```

Comment:
```suggestion
func (srv *Server) monitoredHandler(handler http.Handler, labels string) http.Handler {
```

From reading the code, I see this is always a singular label, so just treat it as such.